File: codes/yunyeong/88_mergesort.py
class Solution:

	def merge(self, nums1: list[int], m: int, nums2: list[int], n: int) -> None:
		if n == 0:
			return None
		for i in range(m, m+n):
			nums1[i] = nums2[m-i]
			nums1.sort()
# 100% beats.

File: codes/yujo/88_mergesort.ts
function merge(nums1: number[], m: number, nums2: number[], n: number): void {
  if (m === 0) {
    nums1.splice(0, n, ...nums2);
    return;
  }

  if (n === 0) {
    return;
  }

  nums1.splice(m, m + n, ...nums2);
  nums1.sort((a, b) => a - b);
}